src/cr/cube/cube.py

```diff
@@ -410,16 +421,47 @@ def _ca_as_0th(self):
     @lazyproperty
     def _cube_dict(self):
         """dict containing raw cube response, parsed from JSON payload."""
+        cube_dict = copy.deepcopy(self._cube_response)
+        # ---cube inflation---
+        # ---In case of numeric arrays, we need to inflate the rows dimension
+        # ---according to the mean subvariables. For each subvar the rows dimension
+        # ---will have a new element related to the subvar metadata.
+        if self._mean_subvariables:
```

Comment:
This entire block would be suited for a separate static method, because it's specific enough. Something like:
```
if self._mean_subvariables:
    dimensions = self._prepare_num_arr_dimensions(dimensions)
```
which would then have a nice docstring, and be properly placed in the code, and not obfuscate the readability of something like `_cube_dict`.